cpp/dolfinx/mesh/Topology.cpp

```diff
@@ -289,22 +290,32 @@ determine_sharing_ranks(MPI_Comm comm, std::span<const std::int64_t> indices)
 /// 2. With undetermined ownership
 /// 3. Not owned by the caller
 std::array<std::vector<std::int64_t>, 2>
-vertex_ownership_groups(std::span<const std::int64_t> cells_owned,
-                        std::span<const std::int64_t> cells_ghost,
+vertex_ownership_groups(std::vector<std::span<const std::int64_t>> cells_owned,
+                        std::vector<std::span<const std::int64_t>> cells_ghost,
```

Comment:
Should this be a `const std::vector<std::span<const std::int64_T>>&`, or is copying of spans very cheap?

Reply:
I am assuming that copying of spans is cheap. I could make it a reference, as you suggest - no harm. Each vector is going to be pretty small.